docs/fxa-engineering/fxa-sub-platform.md

```diff
@@ -0,0 +1,105 @@
+---
+id: fxa-sub-platform
+title: Firefox Accounts Subscription Platform
+sidebar_label: Subscription Platform
+---
+
+## Getting Started
+
+### Pre-Development
+
+To begin working on the subscription platform in the FxA codebase, you will need access to a Stripe account for private and public API developer keys.
+
+If you're a Mozilla employee, you can request a private key from the FxA Subscription Platform team. Otherwise, you can create your own Stripe account to use for testing that is not linked to any bank account information. These keys should be taken from Stripe's test environment.
+
+The `fxa-payments-server` needs the Stripe public key and communicates with the `fxa-auth-server` that requires a Stripe private key. Configuration details can be found below.
+
+### Configuration
+
+You will need to add a `subscriptions.stripeApiKey` field in `fxa/packages/fxa-auth-server/config/dev.json` with the value of your private stripe API key. Ensure the key begins with `pk_test` to guarantee are testing in the correct environment.
+
+Ex:
+
+```
+  "subscriptions": {
+    "enabled": true,
+    "stripeApiKey": "pk_test_####"
+    ...
+```
+
+If you are testing with your own Stripe account, you will need to update the `stripe.apiKey.default` public key set in `fxa/packages/fxa-payments-server/server/config/index.js` to reflect your own. The payments development server _should_ have a default Mozilla public API key in place already but it doesn't hurt to double check.
+
+#### Stripe Product/Plans
+
+To see the available products in the Stripe dashboard, navigate to Billing > Products and click into one of the products to see information about it including the product name, product ID, plan name, plan ID, metadata, logs, and events.
```

Comment:
Perhaps add a comma between "it" and "including"; I was out of breath by the end of that sentence.